### PR TITLE
887 da comparison chart

### PR DESCRIPTION
**Pull request by jackson-harris-iii**

### CODE REVIEW

**Comment by jackson-harris-iii** [comment]
@Ishatt tagging to update you on the status
[screencapture-near-org-data-availability-2023-12-18-13_47_37.pdf](https://github.com/near/near-discovery-components/files/13708858/screencapture-near-org-data-availability-2023-12-18-13_47_37.pdf)
[screencapture-near-org-data-availability-2023-12-18-13_48_01.pdf](https://github.com/near/near-discovery-components/files/13708859/screencapture-near-org-data-availability-2023-12-18-13_48_01.pdf)

File: src/NearOrg/DataAvailabilityPage.jsx [hunk 1]
```diff
@@ -238,6 +387,17 @@ return (
           </Text>
         </Flex>
 
+        {/* This will be uncommented once we get the updated image from the design team */}
```
Comment (charleslavon) [inline src/NearOrg/DataAvailabilityPage.jsx:390]: Are we waiting on another updated image? I'm wondering if this and the other commented code can be removed

**Review by charleslavon — COMMENTED** [review]
Thanks for jumping onto this task @jackson-harris-iii. It will be great to get this merged today, in time for this week's release.

**Comment by jackson-harris-iii** [comment]
> Thanks for jumping onto this task @jackson-harris-iii. It will be great to get this merged today, in time for this week's release.

comments removed!

File: src/NearOrg/DataAvailabilityPage.jsx (hunk 1 above)
Comment (jackson-harris-iii) [reply src/NearOrg/DataAvailabilityPage.jsx]: @charleslavon image updated.

Near Org Data Availability (4)

**Review by charleslavon — APPROVED** [review]

**Pull request merged**